vendor/init.te

```diff
@@ -14,9 +17,14 @@ dontaudit init { sysfs_type proc_type }:file create;
 # Allow the init process to read and write default_smp_affinity:
 allow init proc_irq:file rw_file_perms;
 
+# Mount and read/write to devices
+allow init device:dir { mounton rw_dir_perms };
```

Comment:
same for this ... label the /dev path following caf if they exist ... same for string 23